feat: Platform.sh specific app pull, fixes #5727 (#5728)
ddev/ddev: docs/content/users/providers/platform.md

## Platform.sh Per-Project Configuration

1. Check out the site from Platform.sh and configure it with [`ddev config`](../usage/commands.md#config). You’ll want to use [`ddev start`](../usage/commands.md#start) and make sure the basic functionality is working.

> 2. Add `PLATFORM_PROJECT` and `PLATFORM_ENVIRONMENT` and  `PLATFORM_APP` (optional) variables to your project.

Reviewer: ```suggestion
2. Add `PLATFORM_PROJECT` and `PLATFORM_ENVIRONMENT` variables to your project.
```

    * Either in `.ddev/config.yaml` or a `.ddev/config.*.yaml` file:

        ```yaml
        web_environment:
        - PLATFORM_PROJECT=nf4amudfn23biyourproject
        - PLATFORM_ENVIRONMENT=main

>         - PLATFORM_APP=app (optional)

Reviewer: ```suggestion
```

4. Run `ddev pull platform`. After you agree to the prompt, the current upstream databases and files will be downloaded.
5. Optionally use `ddev push platform` to push local files and database to Platform.sh. The [`ddev push`](../usage/commands.md#push) command can potentially damage your production site, so we don’t recommend using it.

> ### Multiple apps
> It is necessary to specify PLATFORM_APP variable under web_environment if your environment contains more than one app

Reviewer: ````suggestion
### Multiple apps

If your environment contains more than one app, add `PLATFORM_APP` variable to your project:

    * Either in `.ddev/config.yaml` or a `.ddev/config.*.yaml` file:

        ```yaml
        web_environment:
        - ...
        - PLATFORM_APP=app
        ```

    * Or with a command from your terminal:

        ```bash
        ddev config --web-environment-add="PLATFORM_APP=app"
        ```
````
